graphql-server/pkg/queries/proposal.go

```diff
@@ -247,6 +248,24 @@ func (q *ProposalQuery) QueryProposalVotes(proposalIDs []int, addresses []string
 	return votes, nil
 }
 
+func (q *ProposalQuery) QueryProposalVotesByAddress(address string) ([]*models.ProposalVote, error) {
+	query := q.session.NewSelect().
+		Model((*models.ProposalVote)(nil)).
+		Where("voter_address = ?", address)
+
+	count, err := query.Count(q.ctx)
+	if err != nil {
+		return nil, err
+	}
+	votes := make([]*models.ProposalVote, count)
+
+	err = query.Scan(q.ctx, &votes)
```

Comment:
does query Scan work after Count? thought it would consume the query

Reply:
I think it works, the query is able to return the correct vote counts when I do manual testing

Reply:
https://bun.uptrace.dev/guide/query-select.html#count-rows

this exists, and why do you need to count if you won't need it in result? if you want to catch not found just check the err type and handle it